dataactbroker/handlers/accountHandler.py

```diff
@@ -366,6 +432,27 @@ def changeStatus(self,system_email):
         self.interfaces.userDb.changeStatus(user,requestDict.getValue("new_status"))
         return JsonResponse.create(StatusCode.OK,{"message":"Status change successful"})
 
+    def listUsers(self):
+        """ List all users ordered by status. Associated request body must have key 'filter_by' """
+        user = self.interfaces.userDb.getUserByUID(LoginSession.getName(flaskSession))
+        isAgencyAdmin = True if self.interfaces.userDb.hasPermission(user, "agency_admin") else False
+        try:
+            if isAgencyAdmin:
+                users = self.interfaces.userDb.getUsers(agency=user.agency)
+            else:
+                users = self.interfaces.userDb.getUsers()
```

Comment:
non-admins can't use the list_users route. The second case is not for the "current_user". The first case handles if the agency admin is calling the route which returns only users in their agency. The second case is for the website admin which returns all users.